Client/Frontend/Reader/ReaderMode.swift

```diff
@@ -14,6 +14,7 @@ enum ReaderModeState: String {
 enum ReaderModeTheme: String {
     case Light = "light"
     case Dark = "dark"
+    case Print = "print"
```

Comment:
Print is really Sepia. I also think the name is confusing but we can argue about that in a different bug.